src/plugins/crypto.c

```diff
@@ -21,7 +21,7 @@
 #include <glib.h>
 #include <libcryptsetup.h>
 #include <nss.h>
-#include <libvolume_key.h>
+#include <volume_key/libvolume_key.h>
```

Comment:
Ah sorry I missed that bit, pushed an additional commit to remove that to this branch.